life_expectancy/cleaning.py

```diff
@@ -26,16 +30,32 @@ def clean_data(region_name: str):  # pylint: disable=C0116
     data = data[data['value'] != ' ']
     data = data[data['region'] == region_name]
     data['value'] = data['value'].astype(float)
+    return data
+
+
+def save_data(data) -> csv:  # pylint: disable=C0116
     data.to_csv(DIR_PATH / "data" / SAVE_FILE_NAME, index=False)
 
 
 def parse_args():  # pylint: disable=C0116
```

Comment:
Here you can safely remove the pylint disable comment. The function already has a docstring, so it will be fine. 😉